app/src/components/fields/AutocompleteField.tsx

```diff
@@ -27,7 +28,7 @@ const AutocompleteField: React.FC<IAutocompleteField<string | number>> = <T exte
   const { touched, errors, setFieldValue, values } = useFormikContext<IAutocompleteFieldOption<T>>();
 
   const getExistingValue = (existingValue: T): IAutocompleteFieldOption<T> => {
-    const result = props.options.find((option) => existingValue === option.value);
+    const result = props.options.find((option) => existingValue === option[`${props.optionFilter || 'value'}`]);
```

Comment:
Is it possible to avoid using a template string to key the `option`?

```typescript
const result = props.options.find((option) => existingValue === option[props.optionFilter || 'value']);
```

Reply:
yes it is